src/Powercord/plugins/pc-spotify/Modal/Title.jsx

```diff
@@ -11,24 +11,24 @@ module.exports = class Title extends React.Component {
   }
 
   render () {
-    // @todo: make dynamic
-    this.canvas.font = 'normal normal 400 normal 14px / 14px Whitney, "Helvetica Neue", Helvetica, Arial, sans-serif';
+    const titleElement = document.querySelector(`.${this.props.className.replace(/ /g, '.')}`);
+    this.canvas.font = titleElement ? getComputedStyle(titleElement).font : null;
```

Comment:
Back when it was dynamic we got issues because the discrim font properties aren't the same, and causes the text to scroll too far. But since streamer mode wipes the discriminator element we can't compute it blablabla

Ig this is good for now:tm: